apps/condo/pages/news/index.tsx

```diff
@@ -0,0 +1,297 @@
+import { useRouter } from 'next/router'
+import React, { useCallback, useState, useMemo } from 'react'
+
+import { Search, PlusCircle } from '@open-condo/icons'
+import { useIntl } from '@open-condo/next/intl'
+import { useOrganization } from '@open-condo/next/organization'
+import { ActionBar, Typography, Button } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+import Input from '@condo/domains/common/components/antd/Input'
+import {
+    PageWrapper,
+} from '@condo/domains/common/components/containers/BaseLayout'
+import { TablePageContent } from '@condo/domains/common/components/containers/BaseLayout/BaseLayout'
+import LoadingOrErrorPage from '@condo/domains/common/components/containers/LoadingOrErrorPage'
+import { EmptyListView } from '@condo/domains/common/components/EmptyListView'
+import { Table, DEFAULT_PAGE_SIZE } from '@condo/domains/common/components/Table/Index'
+import { TableFiltersContainer } from '@condo/domains/common/components/TableFiltersContainer'
+import { useQueryMappers } from '@condo/domains/common/hooks/useQueryMappers'
+import { useSearch } from '@condo/domains/common/hooks/useSearch'
+import { getPageIndexFromOffset, parseQuery } from '@condo/domains/common/utils/tables.utils'
+import { useTableColumns } from '@condo/domains/news/hooks/useTableColumns'
+import { useTableFilters } from '@condo/domains/news/hooks/useTableFilters'
+import { NewsItem, NewsItemScope } from '@condo/domains/news/utils/clientSchema'
+import { OrganizationRequired } from '@condo/domains/organization/components/OrganizationRequired'
+import { IFilters } from '@condo/domains/ticket/utils/helpers'
+
+interface INewsIndexPage extends React.FC {
+    headerAction?: JSX.Element
+    requiredAccess?: React.FC
+}
+
+const PAGE_ROW_GUTTER: RowProps['gutter'] = [0, 40]
+const SORTABLE_PROPERTIES = ['number', 'createdAt']
+const NEWS_DEFAULT_SORT_BY = ['createdAt_DESC']
+
+
+const NewsTable = ({
+    total,
+    news,
+    columns,
+    loading,
+}) => {
+    return (
+        <>
+            <Table
+                totalRows={total}
+                loading={loading}
+                dataSource={loading ? null : news}
+                columns={columns}
+                data-cy='news__table'
+            />
+        </>
+    )
+}
+
+const NewsTableContainer = ({
+    filterMetas,
+    sortBy,
+    searchNewsQuery,
+    loading,
+}) => {
+    const router = useRouter()
+    const { filters, offset } = useMemo(() => parseQuery(router.query), [router.query])
+    
+    const currentPageIndex = useMemo(() => getPageIndexFromOffset(offset, DEFAULT_PAGE_SIZE), [offset])
+
+    const {
+        loading: isNewsFetching,
+        count: total,
+        objs: news,
+    } = NewsItem.useObjects({
+        sortBy,
+        where: searchNewsQuery,
+        first: DEFAULT_PAGE_SIZE,
+        skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
+    })
+
+    const [whereNewsItemScopeArr, setWhereNewsItemScopeArr] = useState([
```

Comment:
It can be just news id's.
```suggestion
    const newsIds = news.map(obj => obj.id)
    // ...
    const {
      loading: isNewsItemScopeFetching,
      objs: newsItemScope,
    } = NewsItemScope.useObjects({
        where: {
            newsItem: { id_in: newsIds }
        },
    })
```